src/main/java/edu/ie3/datamodel/io/source/couchbase/CouchbaseWeatherSource.java

```diff
@@ -34,7 +34,6 @@
 /** Couchbase Source for weather data */
 public class CouchbaseWeatherSource extends WeatherSource {
   private static final Logger logger = LoggerFactory.getLogger(CouchbaseWeatherSource.class);
-  private static final String DEFAULT_TIMESTAMP_PATTERN = "yyyy-MM-dd'T'HH:mm:ssxxx";
```

Comment:
No strong opinion here, but maybe leave this static field like it is? datetime format Strings are still used in this class.
Or, rather, maybe we also just want a `DateTimeFormatter` required by the constructor, like in `SqlTimeSeriesSource` for example?

Reply:
I don't mind. It was unused. But I'll revert this change